instance/models/instance.py

```diff
@@ -134,6 +152,68 @@ def status(self):
         else:
             return active_server_set[0].status
 
+    @property
+    def event_context(self):
+        """
+        Context dictionary to include in events
+        """
+        return {'instance_id': self.pk}
+
+    @staticmethod
+    def on_post_save(sender, instance, created, **kwargs):
+        """
+        Triggered by the post_save event, with `created` a boolean indicating if the object was just
+        created in the DB
+        """
+        self = instance
+        self.update_logger()
```

Comment:
@antoviaque Ah, I didn't think of that! I checked django docs before reading further, so I know what `instance` means, but yeah I can definitely see why the name `instance` can be confusing :)
The `self = instance` bit is also confusing, though. In the server class you could maybe just name the parameter `server` instead of `instance`:

    def on_post_save(sender, server, created, **kwargs):
        ...